Code/GraphMol/Substruct/SubstructMatch.h

```diff
@@ -57,6 +57,10 @@ struct RDKIT_SUBSTRUCTMATCH_EXPORT SubstructMatchParameters {
                        //!< concurrent threads supported by the hardware
                        //!< negative values are added to the number of
                        //!< concurrent threads supported by the hardware
+  std::vector<std::string> atomProperties = {};  //!< atom properties that must be
+                                                 //!< equivilent in order to match
+  std::vector<std::string> bondProperties = {};  //!< bond properties that must be
+                                                 //!< equivilent in order to match
```

Comment:
empty initialization of vectors is redundant, and a typo fix:
```suggestion
  std::vector<std::string> atomProperties;  //!< atom properties that must be
                                                 //!< equivalent in order to match
  std::vector<std::string> bondProperties;  //!< bond properties that must be
                                                 //!< equivalent in order to match
```